R/mod_query_data.R

```diff
@@ -52,7 +52,7 @@ mod_query_data_ui <- function(id){
               style = "color:#0072B2;",
               title = "At present, TADA is only designed to work with water sample media"
             )
-          ), choices = c("",media), selected = "Water", multiple = TRUE)),
+          ), choices = c("",media), selected = c("Water", "water"), multiple = TRUE)),
```

Comment:
I tried to add "water" to the list of autopopulated media values along with "Water". What else needs to be edited to add "water" here?

Reply:
@cristinamullin It needed to be added to the "media" object in addition to the selectizeInput. Another approach would be to automatically look for "W"ater and "w"ater using the dataRetrieval query if someone picks the media name Water. Which would you prefer?

Reply:
automatically would be nice, but then users of the WQP interface (or if they go back and forth) will not know. So I am torn...

Reply:
We can keep it separated for now. It's less coding that way, haha, and more transparent. I also added in a few other steward/nwis-specific media types.